### PR TITLE
Fixes, single loop and queue loop

### PR DESCRIPTION
**Pull request by SebestikCZ**

### CODE REVIEW

**Comment by SebestikCZ** [comment]
Don't merge at all, I will request your review later

**Comment by SebestikCZ** [comment]
When reviewed, merge

File: cmds/loopqueue.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const { MessageButton, MessageActionRow } = require("gcommands");
+const { MessageEmbed } = require("discord.js");
+
+module.exports = {
+    name: "loop",
```
Comment (xHyroM) [inline cmds/loopqueue.js:5]: It would be better if you put it in one loop command and put modes like /loop mode:queue/song .

**Review by xHyroM — CHANGES_REQUESTED** [review]

File: cmds/loopqueue.js (hunk 1 above)
Comment (SebestikCZ) [reply cmds/loopqueue.js:5]: ok

Comment (SebestikCZ) [reply cmds/loopqueue.js]: I don't have it tested, but I hope, this will work.

Comment (SebestikCZ) [reply cmds/loopqueue.js]: wait much commented on file

Comment (SebestikCZ) [reply cmds/loopqueue.js]: ok good

Comment (SebestikCZ) [reply cmds/loopqueue.js]: and now the replies...

Comment (SebestikCZ) [reply cmds/loopqueue.js]: now good i hope

File: cmds/loop.js [hunk 2]
```diff
@@ -14,15 +14,15 @@ module.exports = {
...
+      let enable = new MessageButton().setLabel("Single Loop").setStyle("blurple").setID(`enableLoop`),
+        disable = new MessageButton().setLabel("Queue Loop").setStyle("blurple").setID(`enableQueueLoop`),
+        cancel = new MessageButton().setLabel("Disable").setStyle("red").setID(`disableLoop`);
       buttonRow.addComponent(enable)
       buttonRow.addComponent(disable)
       buttonRow.addComponent(cancel)
       
       var msg = await respond({
-        content: "• Turn the loop system on/off.",
+        content: "• Turn the loop system queue type/single type/off.",
```
Comment (xHyroM) [inline cmds/loop.js:25]: Do it with `expectedArgs` choices

**Review by xHyroM — CHANGES_REQUESTED** [review]

**Review by xHyroM — COMMENTED** [review]
Please test it before merge.

File: cmds/loop.js (hunk 2 above)
Comment (SebestikCZ) [reply cmds/loop.js:25]: its on your side to change the whole bot to the newest version of gcommands...

**Comment by SebestikCZ** [comment]
Okay, everything tested and working fine, expect slash commands...

**Comment by xHyroM** [comment]
> Okay, everything tested and working fine, expect slash commands...

Fix slash commands

**Comment by SebestikCZ** [comment]
> > Okay, everything tested and working fine, expect slash commands...
> 
> Fix slash commands

But this requires an complete rewrite to the newest version of gcommands...

**Comment by SebestikCZ** [comment]
> > > Okay, everything tested and working fine, expect slash commands...
> > 
> > 
> > Fix slash commands
> 
> But this requires an complete rewrite to the newest version of gcommands...

okay, i'll do that, but it will not be early

**Comment by xHyroM** [comment]
> 
> 
> > > Okay, everything tested and working fine, expect slash commands...
> > 
> > 
> > Fix slash commands
> 
> But this requires an complete rewrite to the newest version of gcommands...

No?

**Comment by xHyroM** [comment]
Slash cmds working?

**Comment by SebestikCZ** [comment]
Last commits are not field tested, please be aware of using it

**Comment by xHyroM** [comment]
good job

**Pull request closed without merging**